docs/contributing/coding-guidelines/ros-nodes/parameters.md

```diff
@@ -1,5 +1,79 @@
+
+```xml
+<include file="$(find-pkg-share tier4_planning_launch)/launch/planning.launch.xml">
+  <arg name="tier4_planning_launch_param_path" value="$(find-pkg-share autoware_launch)/config/tier4_planning_launch"/>
+</include>
+```
+
+### When launching autoware via the component launch
+
+In [`planning.launch.xml`](https://github.com/autowarefoundation/autoware.universe/blob/main/launch/tier4_planning_launch/launch/planning.launch.xml) of the `tier4_planning_launch` package, the default variable for the `tier4_planning_launch_param_path` argument is `$(find-pkg-share tier4_planning_launch)/config`.
+Therefore, parameters in the `tier4_planning_launch` package (`Component Parameter`) are used.
+
+```xml
+<arg name="tier4_planning_launch_param_path" default="$(find-pkg-share tier4_planning_launch)/config" description="tier4_planning_launch parameter path"/>
+```
+
+![parameters-to-be-used](images/parameters-to-be-used.svg){: style="width:1200px"}
+
+## Automatic synchronization of parameters
+
+There is a PR generated automatically to synchronize `Launch Parameter` with `Component Parameter`.
+
+You can see the synchronization setting file [here](https://github.com/autowarefoundation/autoware_launch/blob/main/.github/sync-param-files.yaml).
+Note that all parameter files are listed one by one.
+
+![parameter-sync](images/parameter-sync.svg){: style="width:900px"}
+
+## How to manage parameters
+
+We explain how to manage parameters in some use cases.
+
+### When creating a new package
+
+- Locates the parameters of `Node Parameter` and `Component Parameter` for the new package in `autowarefoundation/autoware.universe`.
+- Create a PR to `autowarefoundation/autoware.universe`.
+- Update [a synchronization setting file](https://github.com/autowarefoundation/autoware_launch/blob/main/.github/sync-param-files.yaml).
+- Sync PR to `autowarefoundation/autoware_launch` is created automatically.
+  - Or you can create this PR manually.
+- Merge both PRs.
+
+### When modifying general parameters
+
+- Modify the parameters of `Node Parameter` and `Component Parameter` in `autowarefoundation/autoware.universe`.
+- Create a PR to `autowarefoundation/autoware.universe`.
+- If you add a new config file, update [a synchronization setting file](https://github.com/autowarefoundation/autoware_launch/blob/main/.github/sync-param-files.yaml).
+- Sync PR to `autowarefoundation/autoware_launch` is created automatically.
+  - Or you can create this PR manually.
+- Merge both PRs.
+
+### When modifying integrated parameters
+
+Note: Assuming that there are your forked repositories of `autowarefoundation/autoware` and `autowarefoundation/autoware_launch`.
+
+This change does not affect the Autoware Foundation's repository.
+Therefore, just modifying your forked repository is fine.
+
+- Modify the parameters of `Launch Parameter` in your forked `autoware_launch` repository.
+- Create a PR and merge.
```

Comment:
Is this what you've intended?
```suggestion
### How to maintain your custom parameters

For example if you want to integrate Autoware with your own vehicles, you may need to tune some parameters in Autoware.

A recommended way is to use `Launch Parameter` in `autowarefoundation/autoware_launch` as your custom parameters. Please fork the repository (e.g. as `YOUR_ACCOUNT/autoware_launch.YOURS`) and modify the parameters as you want.

One advantage of doing so is that the `sync-param-files` GitHub Actions helps you to keep `YOUR_ACCOUNT/autoware_launch.YOURS` up-to-date with `autowarefoundation/autoware.universe`. The GitHub Actions workflow will automatically create sync PR on a regular basis, so you can incorporate the latest changes in `autowarefoundation/autoware.universe` when you want.
```

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/171/commits/9ebdfe40092fcbd08b2741dccbc7dfce0a57afff
DONE

I really appreciate your better expressions!